docs/CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Added
 - Add filtered BAM output to ``macs2-callpeak`` process
 - Add an option to use filtered BAM files from ``macs2-callpeak`` to
   ``rose2``, ``workflow-macs-rose``, and ``macs2-rose2-batch``
+- Add ChIPQC in ``resolwebio/chipseq`` Docker image
```

Comment:
Tag the image version in the changelog, e.g. Add ChIPQC to ``resolwebio/chipseq:4.1.0`` Docker image